docs/source/installation/guide_quick/index.rst

```diff
@@ -0,0 +1,83 @@
+.. _installation:guide-quick:
```

Comment:
This was because I was anticipating that they would have several files, especially for the complete guide. Files for RabbitMQ and PSQL for example. If we end up not using it, I will convert them to single files